README.md

```diff
@@ -64,6 +66,7 @@ For the unsupervised training, we are using a related problem for which we alrea
 - Install the dependencies in requirements.txt: `pip install -r requirements.txt`.
 - Install the dependencies in test-requirements.txt: `pip install -r test-requirements.txt`.
 - Run the **get_dependencies.py** script.
+- Run the **pre-train.py / train.py** script to train the neural network.
```

Comment:
The name of the file is `pretrain.py` and not `pre-train.py`.

You can write this:
```
- Run the **pretrain.py** / **train.py** script to train the neural network.
```